### PR TITLE
Support for Elasticsearch, Redis and Celery as we currently use them

### PR DESCRIPTION
**Pull request by giocalitri**
Updated the app to support the Elasticsearch, Redis and Celery as we currently use them.
In particular Celery required a change of settings variable names.
Updated also how the code is tested and how the docker container works

The actual changes in functionality are only in `views.py`

### CODE REVIEW

File: server_status/tests/settings.py [hunk 1]
```diff
@@ -11,32 +11,89 @@
...
+COVERAGE_MODULE_EXCLUDES += EXTERNAL_APPS
+
+SECRET_KEY = 'foobar'
+
+ELASTICSEARCH_URL = os.environ.get('ELASTICSEARCH_URL', 'http://127.0.0.1:9200')
+
+USE_CELERY = True
+STATUS_TOKEN = 'asdf'
+
+BROKER_URL = os.environ.get('BROKER_URL', 'redis://127.0.0.1:6379/4')
```
Comment (blarghmatey) [inline server_status/tests/settings.py]: Is there a particular reason that the broker URL is defaulting to database 4 in this case? My understanding is that Redis DBs usually start at 0.

File: server_status/views.py [hunk 2]
```diff
@@ -69,12 +68,14 @@ def get_redis_info():
...
-    except AttributeError as ex:
-        log.error("No redis connection info found in settings. Error: %s", ex)
+    for conf_name in ('CELERY_BROKER_URL', 'BROKER_URL', 'REDIS_URL', ):
+        if hasattr(settings, conf_name):
+            url = getattr(settings, conf_name)
+            break
+    else:
+        log.error("No redis connection info found in settings.")
         return {"status": NO_CONFIG}
+    _, host, port, _, password, database, _ = parse_redis_url(url)
```
Comment (blarghmatey) [inline server_status/views.py]: This is hardcoding the assumption that the Celery broker will only ever be Redis.

Comment (giocalitri) [inline server_status/views.py]: well... yeah, but my idea was not to add support for different brokers, just restore the functionality already implemented.

File: server_status/tests/settings.py (hunk 1 above)
Comment (giocalitri) [inline server_status/tests/settings.py]: I left database in the URLs like they were: I can change to 0

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**